lib/frame-manager.js

```diff
@@ -42,10 +41,15 @@ function FrameManager(window) {
     if (!self.listenerCount('data')) {
       parent.emit('log', 'unsubscribing from browser window frames')
       window.webContents.endFrameSubscription();
-      subscribed = false;
     }
   }
 
+  function cancelFrame() {
+    requestedFrame = false;
+    clearTimeout(frameRequestTimeout);
+    self.emit('data', null);
+  }
+
```

Comment:
Oh, I totally missed that first part. That's what I get for reading it half-awake x_x